main.go

```diff
@@ -139,6 +139,44 @@ func init() {
 }
 
 func main() {
+	var (
+		nodeClient     chef.Client
+		ohaiJSON       = map[string]interface{}{}
+		convergeJSON   = map[string]interface{}{}
+		complianceJSON = map[string]interface{}{}
+	)
+
+	if config.RunChefClient {
+		nodeClient = getAPIClient(config.ClientName, config.ClientKey, config.ChefServerURL)
+	}
+
+	if config.OhaiJSONFile != "" {
+		ohaiJSON = parseJSONFile(config.OhaiJSONFile)
+	}
+	if config.ConvergeStatusJSONFile != "" {
+		convergeJSON = parseJSONFile(config.ConvergeStatusJSONFile)
+	}
+
+	if config.ComplianceStatusJSONFile != "" {
+		complianceJSON = parseJSONFile(config.ComplianceStatusJSONFile)
+	}
+
+	if config.RandomData {
+		// TODO: (@afiune) Re design a bit more to have different use-cases, maybe sub-commands?
+		//
+		// 1) Start the Chef-Load service for continous data:
+		//       chef-load start -config foo.conf
+		// 2) Load one time data with random fields:
+		//       chef-load generate 100 -config foo.conf
+		//
+		// For now we just have one flag -random to land in this if
```

Comment:
Where is the -random flag defined? I only see a boolean RandomData being added to the config, not how it's getting activated by a commandline flag.

Reply:
I didn't add it because of the "idea" I explained above about the sub-commands. I can add it though